### DISCUSSION TITLE
Undocumented `#monthPattern` attribute

### DISCUSSION THREAD

**Discussion by Compositr** [TriliumNext/Trilium#4253] [Ideas] [answered]
Hey everyone,
I've noticed that, while `#datePattern` is [documented](https://github.com/zadam/trilium/wiki/Day-notes#date-pattern), `#monthPattern` on the other hand isn't. It would be great if it was documented!

Code for `#monthPattern` implementation: https://github.com/zadam/trilium/blob/c881b398605f5dbd3eed7345c673e167055d825a/src/services/date_notes.js#L96-L103

**Comment by Compositr** [comment, marked as answer]
Here is my suggested addition to the existing docs page:

# Month pattern
It is also possible to customize the title of generated month notes through the `#monthPattern` attribute, much like `#datePattern`. The options are:
- `{monthNumberPadded}` results in a number like `09` for September, and `11` for November
- `{month}` results in the full month name (e.g. `September` or `October`)

The default is `{monthNumberPadded} - {month}`

```md

# Month pattern
It is also possible to customize the title of generated month notes through the `#monthPattern` attribute, much like `#datePattern`. The options are:
- `{monthNumberPadded}` results in a number like `09` for September, and `11` for November
- `{month}` results in the full month name (e.g. `September` or `October`)

The default is `{monthNumberPadded} - {month}`
```

**Reply by zadam** [reply]
I've added your text into docs, thanks!
